### DISCUSSION TITLE
After reboot K8S hosts, skywalking-opa kept reporting errors: table alarm_record does not exist. OAP is running in 'no-init' mode, waiting... retry 3s later.

### DISCUSSION THREAD

**Discussion by zhilong-rancher** [apache/skywalking#8670] [Q&A]
I deployed Skywalking version 8.9.1 in K8S, using ES 6, with data persistence.

When K8S was running properly, I deleted the POD of ES and did not delete the persistent volume. After a while, ES Pod was rebuilt and SkyWalking was available.

However, when I shut down all the K8S hosts and started the server again, K8S returned to normal and the POD of ES also returned to normal. The POD of Skywalk-OPA could not be started and kept reporting errors:
`table alarm_record does not exist. OAP is running in 'no-init' mode, waiting... retry 3s later.`

I tried to re-execute the Skywalk-es-init job, and get `OAP starts up in init mode successfully, exit now...`,but the problem still there.

**Comment by wu-sheng** [comment]
> I deleted the POD of ES and did not delete the persistent volume

If this is true, why indices don't exist? Have you checked your ElasticSearch server?

**Reply by zhilong-rancher** [reply]
Some additional information: My ES uses NFS storage and the NFS service is provided by one of the K8S hosts.

I'm talking about two scenarios. The first one will not affect SkyWalking usage, and the second one will cause SkyWalking OAP to fail to start.

1：When K8S was running properly, I deleted the POD of ES and did not delete the persistent volume. After a while, ES Pod was rebuilt and SkyWalking OAP return to normal. And no data was lost，I can keep using it。
I check the indexes in ES by http://es-server/_cat/indices?v , and can see that there are some indices named  sw_alarm_record-YYYYMMDD ,no other index contains the word alarm_record.

2：When I shut down all the K8S hosts and started the server again, K8S returned to normal and the POD of ES also returned to normal. The POD of Skywalking OAP could not be started and kept reporting errors:
`table alarm_record does not exist. OAP is running in 'no-init' mode, waiting... retry 3s later.`
I check the indexes in ES by http://es-server/_cat/indices?v , the same data as SkyWalking was up and running, and can see that there are some indices named  sw_alarm_record-YYYYMMDD ,no other index contains the word alarm_record.

**Reply by wu-sheng** [reply]
Case (2) is generally relative to how you manage the volume of pods, SkyWalking has nothing to do the whole env scope.
